README.md

```diff
@@ -53,19 +53,19 @@ CentOS versions currently supported are:
 
 ## Installation
 
-Choose either the CentOS Stream 9 or RHEL9 based image:
+Choose either the CentOS Stream 9 or RHEL9-based image:
 
 - **RHEL9 based image**
 
   These images are available in the [Red Hat Container Catalog](https://access.redhat.com/containers/#/registry.access.redhat.com/rhel9/postgresql-13).
-  To download it run:
+  To download it, run the following:
 
   ```
   podman pull registry.redhat.io/rhel9/postgresql-13
   ```
 
-  To build a RHEL9 based image, you need to run Docker build on a properly
-  subscribed RHEL machine.
+  To build an RHEL9-based image, you need to run Docker build on a properly
```

Comment:
```suggestion
  To build a RHEL9-based image, you need to run Docker build on a properly
```
I would say, the 'a' was correct here, wasn't it?

Reply:
Copy-paste mistake. Thank you!